### PR TITLE
Upgrade angular 1.5.3

### PR DESCRIPTION
**Pull request by Minglee01**
@incuna/js please review.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -25,8 +25,8 @@
     "api"
   ],
   "dependencies": {
-    "angular": "~1.2",
-    "angular-ext-q": "~1.0",
+    "angular": "1.2 - 1.5",
+    "angular-ext-q": "1.1.0",
```
Comment (grahamgilchrist) [inline bower.json]: @Minglee01 I don't think we should pin this to `1.1.0`. Using `~1` will still match `1.1.0` but also update to any other `1.x` version.

Comment (henrahmagix) [inline bower.json]: Perhaps we could move away from using ~ `~1` and use x `1.x`

**Comment by grahamgilchrist** [comment]
Looks good!

**Comment by henrahmagix** [comment]
These kinds of merges would be great to be squashed using GitHub's new merge feature: we don't need 8 commits for a version upgrade

**Pull request merged**